Gemfile

```diff
@@ -21,8 +21,7 @@ gem 'haml'
 gem 'htmlentities'
 gem 'kaminari'
 gem 'kaminari-mongoid'
-gem 'mongoid', '~> 5.4'
-# gem 'mongoid', '6.0.3'
+gem 'mongoid', '6.0.3'
```

Comment:
Can we make this more flexible? like ~> 6.0?

Reply:
@stevecrozz this gem update fills fragile. So, it locked for future. Anyway, 6.0.3 is last version that works with rails 5.0.

I hope, I answer all questions?